src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Authentication.cs

```diff
@@ -132,6 +132,13 @@ public abstract class HttpClientHandler_Authentication_Test : HttpClientTestBase
 
         public static IEnumerable<object[]> Authentication_TestData()
         {
+            yield return new object[] { "Basic realm=\"testrealm1\" basic realm=\"testrealm1\"", true };
```

Comment:
It would helpful to add comments to these test cases to explain each one in terms of valid/invalid and what the test case is trying to prove/disprove.

This would make it easier and faster to understand what is being tested and whether or not we have gaps in the test cases. This is especially important when other devs look at this that might not have networking experience, etc.

Reply:
It looks like I have other changes to make too, but I will definitely add this.